pkg/kube/cr_manager.go

```diff
@@ -18,6 +20,11 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	clusterRoleStarboard        = "starboard"
+	clusterRoleBindingStarboard = "starboard"
+)
+
 const (
```

Comment:
Can you get rid of this `polarisConfigYAML` now there is the ConfigMap?

Reply:
I'd love to get rid of duplicate struct and string literals in favour of YAML files embedded in starboard executable (see https://github.com/golang/go/issues/41191) So for now the question is whether we want to get rid of starboard init command and use kubectly apply as the init step. I thought that these are alternative ways to initialize starboard. WDYT?

Reply:
oh, I see... I think `starboard init` is clearer and easier than having to manually do an `apply`